djgentelella/management/commands/loaddevstatic.py

```diff
@@ -155,6 +155,21 @@ def handle(self, *args, **options):
             'chartjs':[
                 'https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.min.js',
                 'https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.min.css'
+            ],
+
+            'mjolnic-bootstrap-colorpicker': [
+                'https://colorlib.com/polygon/vendors/mjolnic-bootstrap-colorpicker/dist/css/bootstrap-colorpicker.min.css',
+                'https://colorlib.com/polygon/vendors/mjolnic-bootstrap-colorpicker/dist/js/bootstrap-colorpicker.min.js'
+            ],
+            "img/": [],
```

Comment:
this item is not allowed to be included

Reply:
It is like a kind of hack to create that directory without making more significant changes to the code. I am thinking how to do it in the correct way.